src/components/Grants/CurrentGrantsBanner.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useMemo } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { isEmpty } from 'lodash'
+
+import { GrantWithUpdateAttributes } from '../../entities/Proposal/types'
+import { numberFormat } from '../../modules/intl'
+
+import Banner, { BannerType } from './Banner'
+
+const getBannerStats = (grants: GrantWithUpdateAttributes[]) => {
+  if (isEmpty(grants)) {
+    return {}
+  }
+
+  const releasedValues = grants.map((item) => {
+    if (item.configuration.tier === 'Tier 1' || item.configuration.tier === 'Tier 2') {
```

Comment:
`configuration.tier` has type `ProposalGrantTier`
![image](https://user-images.githubusercontent.com/45410089/179509412-c66dbc4e-b785-4508-9939-d8687aa35f04.png)

Reply:
ProposalGrantTier has the full text tier 🤦‍♂️, and texts like 'Tier 1' and 'Tier 2' are coming from the transparency data. Maybe we can have an enum for the categories coming from transparency...